SnoopCompileCore/src/snoopi_deep.jl

```diff
@@ -1,3 +1,5 @@
+export @snoopi_deep
```

Comment:
Were you going to alias `@snoopi_deep` as `@snoopi`?

Reply:
In a separate PR, yes. Let's keep this one focused on deleting stuff.